backend/uclapi/roombookings/token_auth.py

```diff
@@ -9,18 +8,22 @@ def wrapped(request, *args, **kwargs):
         token = request.GET.get("token")
 
         if not token:
-            return JsonResponse({
+            response = JsonResponse({
                 "ok": False,
                 "error": "No token provided"
             })
+            response.status_code = 400
```

Comment:
I think this should a `401` 👉  [Internet Engineering Task Force (IETF)](https://tools.ietf.org/html/rfc7235#section-3.1)